spec/recipes/minimize_access_spec.rb

```diff
@@ -27,56 +27,62 @@
     stub_command("find /bin  -perm -go+w -type f | wc -l | egrep '^0$'").and_return(false)
   end
 
-  let(:chef_run) do
+  cached(:chef_run) do
     ChefSpec::ServerRunner.new.converge(described_recipe)
   end
 
+  subject { chef_run }
+
   it 'remove write permission from /usr/local/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/sbin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/sbin'
+    ).with(
       command: 'chmod go-w -R /usr/local/sbin'
     )
   end
 
   it 'remove write permission from /usr/local/bin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/bin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/bin'
+    ).with(
       command: 'chmod go-w -R /usr/local/bin'
     )
   end
 
   it 'remove write permission from /usr/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/sbin').with(
+    is_expected.to run_execute('remove write permission from /usr/sbin').with(
```

Comment:
@ncs-alane I did not know/notice its disabled

Yes, feel free to enable it, I usually have it like this

```
Metrics/LineLength:
  Max: 120
```

This should be probably Ok, what do you think?

I took a look to the `.rubocop.yml` in chef-os-hardening and chef-ssh-hardening, maybe it makes sense to go through this file in both repos in a dedicated PRs and see if we can minimize the configured settings and have more defaults from rubocop